irods_capability_automated_ingest/sync_irods.py

```diff
@@ -174,11 +167,11 @@ def upload_file_from_S3(logger, session, meta, src, dest, offset=0, **options):
 
     #Could pass in as parameter when running ingest job, keep in meta
     #May be needed for files uploaded to AWS S3 with multi-part upload
-    BUFFER_SIZE = 1024
+    BUFFER_SIZE = 1024*1024
     client = get_s3_client(meta)
     path_list = src.lstrip('/').split('/', 1)
     if len(path_list) == 1:
-        logger.error("failed", task="irods_upload_S3file", path=src)
+        raise ValueError("Incorrect path to S3 object. Object name or bucket name missing.")
```

Comment:
Maybe include src in this as well so the user can tell which path was busted?
```suggestion
        raise ValueError("Object name or bucket name missing from path to S3 object: [{}]".format(src))
```